frontend/src/metabase/visualizations/visualizations/Table.jsx

```diff
@@ -376,11 +378,31 @@ export default class Table extends Component {
     return getTitleForColumn(cols[columnIndex], series, settings);
   };
 
+  getColumnSortDirection = columnIndex => {
```

Comment:
Follow the same pattern as with `getColumnTitle`, where the function accepts `columnIndex`